docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,28 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+[Upcoming Feature] For the Soroban RPC, API endpoints will provide JSON format to unpack XDR base64 strings as long as a flag for `xdrFormat: "json"` is passed to the endpoints.
```

Comment:
Can we just add this when it's ready :joy: :pray:

Reply:
Okie dokie